base/Enums.jl

```diff
@@ -29,7 +29,7 @@ end
 """
     @enum EnumName EnumValue1[=x] EnumValue2[=y]
 
-Create an [`Enum`](:obj:`Enum`) type with name `EnumName` and enum member values of
+Create an `Enum` type with name `EnumName` and enum member values of
```

Comment:
Similarly for UInt, Int, Float16 types. I think seems like this could be fixed in a later PR. Linking every `` `type_name` `` automatically seems like a neat Documenter feature!